commercetools-models/src/main/java/io/sphere/sdk/categories/commands/updateactions/RemoveAsset.java

```diff
@@ -0,0 +1,20 @@
+package io.sphere.sdk.categories.commands.updateactions;
+
+import io.sphere.sdk.categories.Category;
+import io.sphere.sdk.commands.UpdateActionImpl;
+
+/**
+ * Removes an asset.
+ */
+public final class RemoveAsset extends UpdateActionImpl<Category> {
+    private final String assetId;
+
+    public RemoveAsset(final String assetId) {
```

Comment:
Same as `ChangeAssetOrder`: constructor private and creation via static constructor.